infrastructure/metadata/infrastructure/testdriver/actions/__dir__.ini

```diff
@@ -0,0 +1,2 @@
+disabled:
+  if product == "chrome" and os == "mac": true # https://bugs.chromium.org/p/chromedriver/issues/detail?id=3114
```

Comment:
`if product == "chrome" and os == "mac": https://bugs.chromium.org/p/chromedriver/issues/detail?id=3114`

Any value is truthy, and tooling here doesn't always preserve comments.

Reply:
Updated. There's also a nifty "Insert a suggestion <ctrl+g>" button you can use when you know exactly what the edit should be.